src/tests/ftest/dfuse/caching_check.py

```diff
@@ -9,6 +9,22 @@
 from ior_utils import IorCommand, IorMetrics
 from general_utils import pcmd
 
+def percent_change(val1, val2):
```

Comment:
Can we put this in general_utils.py. Can be used by other tests as well easily.

Reply:
Yes, agree. Thanks!